tests/MongoDBBSONBinary_set_state-001.phpt

```diff
@@ -0,0 +1,48 @@
+--TEST--
+MongoDB\BSON\Binary::__set_state()
+--FILE--
+<?php
+$tests = [
+    ['foobar', MongoDB\BSON\Binary::TYPE_GENERIC],
+    ['', MongoDB\BSON\Binary::TYPE_GENERIC],
+    ["\0foo", MongoDB\BSON\Binary::TYPE_GENERIC],
+    [hex2bin('123e4567e89b12d3a456426655440000'), MongoDB\BSON\Binary::TYPE_UUID],
+    [md5('foobar', true), MongoDB\BSON\Binary::TYPE_MD5],
+];
+
+foreach ($tests as $test) {
+    list($data, $type) = $test;
+
+    var_export(MongoDB\BSON\Binary::__set_state([
+        'data' => $data,
+        'type' => $type,
+    ]));
+    echo "\n\n";
+}
+
+?>
+--EXPECT--
+MongoDB\BSON\Binary::__set_state(array(
+  'data' => 'foobar',
+  'type' => 0,
+))
+
+MongoDB\BSON\Binary::__set_state(array(
+  'data' => '',
+  'type' => 0,
+))
+
+MongoDB\BSON\Binary::__set_state(array(
+  'data' => '' . "\0" . 'foo',
+  'type' => 0,
+))
+
+MongoDB\BSON\Binary::__set_state(array(
+  'data' => '>Eg�ӤVBfUD' . "\0" . '' . "\0" . '',
```

Comment:
Are you missing control characters here? When I opened a PR with my Binary tests, GitHub rendered the file as binary and refused to show a diff.